### ISSUE TITLE
`compute_params_filterbank` should return `xis` and `sigmas` but not `js`

### ISSUE THREAD

**Issue by lostanlen** [kymatio/kymatio#856]
came up in conversation with @MuawizChaudhary 

A sequel of #847 : `compute_params_filterbank` should be simpler
At the moment, this function returns three output values: `xis`, `sigmas`, and `js`

But it turns out that every `j` can be derived from `xi` and `sigma` by means of 
 `j = get_max_dyadic_subsampling(xi, sigma, alpha)`
 
Which we could have in the `scattering_filter_factory` itself.
This will avoid carrying over a term back-and-forth between the "factory" and the "params"

in hindsight, this is also beneficial for distinguishing wavelet params (xis, sigmas) from their sampling (js).

**Comment by janden** [comment]
Could work… Would this trickle down to `meta` as well? There we may want to be as verbose as possible given that the users may have different needs.

**Comment by lostanlen** [comment]
>  Could work… Would this trickle down to meta as well? There we may want to be as verbose as possible given that the users may have different needs.

Yes, `self.meta()` would still have the `j`'s. I have a plan for this.

**Comment by lostanlen** [comment]
my plan is implemented in #893
